src/pages/projects-page/ProjectExpansionPanel.tsx

```diff
@@ -24,18 +27,42 @@ type Props = FCProps<OwnProps>;
 export function ProjectExpansionPanel(props: Props) {
   const { initialProjectIds, saveProject, deleteProject, users } = props;
   const projectReview = useFragment(fragment, props.projectReview);
+  const [isExpanded, setIsExpanded] = React.useState(() => !initialProjectIds.has(projectReview.id));
+  const {
+    dialog: { onClose, open },
+    button: { onClick },
+  } = useDialog(false);
+
+  const handleShowModal = (e: React.MouseEvent<HTMLButtonElement>) => {
+    e.stopPropagation();
+    onClick();
+  };
+
+  const handleExpanded: ExpansionPanelProps['onChange'] = (e, expanded) => {
+    setIsExpanded(expanded);
+  };
 
   return (
-    <FormChangeDetector>
-      <ExpansionPanel defaultExpanded={!initialProjectIds.has(projectReview.id)}>
-        <ExpansionPanelSummary>
-          <Typography variant="h6">{projectReview.projectName}</Typography>
-        </ExpansionPanelSummary>
-        <ExpansionPanelDetails>
-          <ProjectForm onSubmit={saveProject} onDelete={deleteProject} projectReview={projectReview} users={users} />
-        </ExpansionPanelDetails>
-      </ExpansionPanel>
-    </FormChangeDetector>
+    <React.Fragment>
+      <FormChangeDetector>
+        <ExpansionPanel expanded={isExpanded} onChange={handleExpanded}>
```

Comment:
What about `defaultExpanded` ?

Reply:
we controlled `ExpansionPanel`.
When we pass `expanded`, 'defaltExpanded` ignored.